src/test/scala/org/kibanaLoadTest/simulation/generic/GenericJourney.scala

```diff
@@ -59,26 +59,27 @@ object ApiCall {
       if (request.headers.contains("Kbn-Version"))
         defaultHeaders + ("Kbn-Version" -> config.version)
       else defaultHeaders
+    val path = request.path.replaceAll(".+?(?=\\/bundles)", "");
```

Comment:
Since from build to build full path will have different prefix (sequence of digits), I think it is better to cut digits part and make it consistent in report. thoughts?
```
> GET /bundles/plugin/kibanaOverview/kibana/kibanaOverv... (OK=17     KO=0     )
> GET /bundles/plugin/newsfeed/kibana/newsfeed.plugin.j... (OK=17     KO=0     )
> GET /bundles/plugin/ux/8.0.0/ux.plugin.js                (OK=17     KO=0     )
```

Alternatively, the full one is `/9007199254740991/bundles/plugin/security/8.0.0/security.plugin.js`

Reply:
Your suggestion makes sense to me as it keeps paths consistent over time. We could probably pick a different name for `path` to convey that this is only used for reporting and not the actual full request path used on the wire.